resharper/resharper-unity/src/Rider/UnitTesting/RunViaUnityEditorStrategy.cs

```diff
@@ -133,6 +145,10 @@ Task IUnitTestRunStrategy.Run(IUnitTestRun run)
                         if (!result.Result)
                         {
                             tcs.SetException(new Exception("There are errors during compilation in Unity."));
+                            
+                            var notification = new RdNotificationEntry("Compilation failed", "Scripts compilation in Unity failed, so tests were not started.", true, RdNotificationEntryType.INFO);
+                            mySolution.Locks.ExecuteOrQueueEx(run.Lifetime, "RunViaUnityEditorStrategy compilation failed", () => myNotificationsModel.Notification.Fire(notification));
+                            myUnityHost.PerformModelAction(model => model.ActivateUnityLogView.Fire());
```

Comment:
Minor nitpick: "Script compilation" instead of "scripts".

And move `notification` inside the lambda passed to `ExecuteOrQueueEx`. It's a ridiculously tiny optimisation, and won't make any noticeable difference, but I've got to comment on something :)